cli/main/main.go

```diff
@@ -33,7 +32,7 @@ func main() {
 	app.Author = "Skippbox Kompose Contributors"
 	app.Email = "https://github.com/skippbox/kompose"
 	app.EnableBashCompletion = true
-	app.Before = cliApp.BeforeApp
+	app.Before = command.BeforeApp
 	app.Flags = append(command.CommonFlags())
 	app.Commands = []cli.Command{
 		command.ConvertCommand(),
```

Comment:
in `command.BeforeApp` you are counting on ConvertCommand being always in the first place here.
If you want it like that, here should be comment explaining that this cannot be moved.